test/distributed/org/apache/cassandra/distributed/test/accord/AccordTestBase.java

```diff
@@ -439,12 +439,14 @@ private static boolean isIdempotent(Cluster cluster, String cql)
         });
     }
 
-    private static String wrapInTxn(String statement)
+    protected static String wrapInTxn(String statement)
     {
         if (!statement.trim().toUpperCase().startsWith("BEGIN TRANSACTION"))
         {
             statement = statement.trim();
-            statement = Arrays.stream(statement.split("\\n")).collect(Collectors.joining("\n  ", "BEGIN TRANSACTION\n  ", "\nCOMMIT TRANSACTION"));
+            statement = Arrays.stream(statement.split("\\n"))
+                              .map(line -> line.trim().endsWith(";") ? line : line + ';')
```

Comment:
requiring me to always do `;` got annoying... just saying....

Reply:
I think the historical reason for this is that without it, there was some ambiguity in the Antlr grammar. That may or may not still be the case...

Reply:
we have different methods... this one is a single `String` and the most common case is a single statement... so why force `;` in those cases?  If you have multiple statements in this string we still split by `\n`, but it gets weird looking... for those cases its far more common to use a different method that takes `String...`